src/linear_algebra/basic.lean

```diff
@@ -2213,3 +2213,29 @@ rfl
 end general_linear_group
 
 end linear_map
+
+section
+
+variables [ring R] [add_comm_group M] [module R M] (T : M →ₗ[R] M)
+
+/-- any invertible linear map can be written as a linear equivalence -/
+def linear_equiv.of_invertible [invertible T] : M ≃ₗ[R] M :=
+linear_map.general_linear_group.to_linear_equiv (unit_of_invertible T)
+
+lemma linear_equiv.coe_of_invertible [invertible T] :
+  ⇑(linear_equiv.of_invertible T) = T := rfl
+
+lemma linear_equiv.coe_linear_map [invertible T] :
+  ↑(linear_equiv.of_invertible T) = T := by { ext, refl }
+
+lemma linear_map.inv_of_eq_symm_of_invertible
+  [invertible T] : ⅟T = (linear_equiv.of_invertible T).symm := by { ext, refl }
```

Comment:
I think this should be stated in reverse, and be `simp`.

Reply:
done